platform/source/lib/ax/platform/minion_manager/cloud_provider/aws/aws_bid_advisor.py

```diff
@@ -278,13 +278,14 @@ def get_spot_instance_price(self, instance_type, zone):
         """
```

Comment:
Write a comment that caller needs to acquire a specific lock before making this call.